client/client.go

```diff
@@ -121,3 +123,110 @@ func (dfs *DistributedFileSystem) GetAttributes(path string) (common.FileAttribu
 		return common.FileAttributes{}, false
 	}
 }
+
+// TODO: return error?
```

Comment:
Although it may be easy to guess, I suggest adding comments about the meaning of the return value.

Reply:
You can return an error or a bool to indicate success/failure. Both are ok. If you have an error message that will be great.